### ISSUE TITLE
Add option to use containerd instead of dockerd

### ISSUE THREAD

**Issue by cronventis** [DockStation/dockstation#302]
It would be awesome to have DockStation working with containerd directly instead of only dockerd.
This would make it the only alternative for docker desktop if you switch to something else completely.

**Comment by igor-lemon** [comment]
Open `Containers` tab
![image](https://user-images.githubusercontent.com/14764799/148520820-dd963019-ea89-41a3-b039-7c49e27c592e.png)

**Comment by cronventis** [comment]
![image](https://user-images.githubusercontent.com/60728024/148526922-4328464f-c83d-4d3f-81a9-3641d75b3057.png)

**Comment by igor-lemon** [comment]
Do you run Docker via Docker machine?

**Comment by cronventis** [comment]
No, I dont run Docker or Docker machine, I run containerd with nerdctl :)
Thats what my request is about.
